README.md

```diff
@@ -347,6 +347,17 @@ CLA signing still works, but has been disabled in the public site in favor of [t
 * cla
 * join_ccla
 
+### Fieri Feature
```

Comment:
These should be in the docs site fieri config section and not in the readme

Reply:
did the changes accordingly